test/util/core/called-in-order-test.js

```diff
@@ -33,13 +33,13 @@ describe("util/core/calledInOrder", function () {
         it("returns true, if stubs were called in given order", function () {
             assert(calledInOrder([testObject1.someFunction, testObject2.otherFunction]));
             assert(calledInOrder([testObject1.someFunction, testObject2.otherFunction,
-                                        testObject3.thirdFunction]));
```

Comment:
I agree with the changes in this file, but they add noise. In the future, someone is going to land on this commit using `git bisect`, and the clearer each commit is, the faster they can get on with their day.

Would you mind splitting this commit, and put the white-space only changes in this file into a separate commit?